event/event.go

```diff
@@ -95,6 +95,7 @@ var (
 	TargetTypeGlobal          = TargetType("global")
 	TargetTypeApp             = TargetType("app")
 	TargetTypeJob             = TargetType("job")
+	TargetTypeCronJob         = TargetType("cronjob")
```

Comment:
I suggest to not have other kind, please use CustomData to differ job from cronjob

Reply:
I disagree, this is just to group the events by it's appropriate "owner" type, the user will want to list the events by the CronJob name and if we use the "Job" directive for everything it will create ambiguity among users.
Furthermore, both types currently exist in this current tsuru build, so it makes perfect sense to differentiate from them in `job list`.

Reply:
semantically we have just a resource job in restapi, another name could be confuses the user.